reddit/views.py

```diff
@@ -126,10 +147,14 @@ def comments(request, thread_id=None):
             pass
 
     return render(request, 'public/comments.html',
-                  {'submission': this_submission,
-                   'comments': thread_comments,
-                   'comment_votes': comment_votes,
-                   'sub_vote': sub_vote_value})
+                    {
+                        'submission': this_submission,
+                        'comments': thread_comments,
+                        'comment_votes': comment_votes,
+                        'sub_vote': sub_vote_value,
+                        'is_user_admin': is_user_admin
+                    }
+                )
```

Comment:
Hi @NkosanaKG can you have a look at this please and make sure my implementation is correct